etl/src/main/java/org/pih/hivmigration/etl/sql/StagingDataMigrator.groovy

```diff
@@ -0,0 +1,115 @@
+package org.pih.hivmigration.etl.sql
+
+class StagingDataMigrator extends SqlMigrator {
+
+    void migrate() {
+        executeMysql("Create hivmigration_health_center table", '''
+            create table hivmigration_health_center (
+            	hiv_emr_id int PRIMARY KEY,
+                openmrs_id int NOT NULL
+            );
+            
```

Comment:
I know this is just copying what was already in place, but we should be careful about this.  In particular, this is not resilient to location name changes, and will silently just put null in for that location.  We should also review and ensure this matches what is actually in the HIV EMR (and used).  The raw data is:

INSTITUTION_ID,NAME
122,
142,Jean Denis
82,Hopital Saint Nicolas de Saint Marc (HSN)
3,Boucan-Carré
4,Cange
5,Belladères
6,Hinche
7,Thomonde
8,Lascahobas
21,Cerca-la-Source
22,Saint-Marc SSPE
31,Cerca cavajal
32,Thomassique
33,Mirebalais
34,Savanette
35,Baptiste
36,
39,Dufailly
38,Tilory
40,Promotion Objectif Zerosida (POZ)
37,Petite Rivière
42,Autre (non-ZL)
63,Verrettes
162,Thomassique
163,Cerca Cavarjal
123,Jean Denis

Reply:
All lines up except 36, which the code has as "TODO Maissade."

Maybe I should make these refer to location UUID instead of name?

Or just validate that locations are non-null? Seems like it would be pretty easy to add some programmatic validation steps to the pipeline.

Reply:
I haven't dove deeply into the exact question here, but, yeah, I like the idea of programmatic validation as needed.  Like maybe assertNotNull(columnName) which makes sure there are no null values in a specific column?